service/src/test/java/uk/gov/hmcts/reform/fpl/service/DraftCMOServiceTest.java

```diff
@@ -156,8 +156,8 @@ void shouldReturnCaseManagementOrderWhenProvidedCaseDetails() {
 
         caseData.put(HEARING_DATE_LIST.getKey(), getDynamicList());
 
-        CaseManagementOrder caseManagementOrder = draftCMOService.prepareCMO(
-            mapper.convertValue(caseData, CaseData.class), null);
+        CaseManagementOrder caseManagementOrder = draftCMOService.prepareCaseManagementOrder(
```

Comment:
New code does something with the documents and hearing details (conditionally / clears up the docs), shouldn't we have test cases for those scenarios?

Reply:
yep this needs to be added. I also have opened this tech debt ticket, as this test is very very lightweight for a heavy method. I will only add assertions around these two new fields, and other tests can be added in https://tools.hmcts.net/jira/browse/FPLA-1721